### PR TITLE
JS-Syntax: Expressions and Statements

### PR DESCRIPTION
**Pull request by YvongYang**

### CODE REVIEW

File: js-syntax/02.JavaScript-Expressions and Statements.md [hunk 1]
```diff
@@ -0,0 +1,1138 @@
...
+            >  var a = 2, b = 3; // 多个变量的初始化
+			> ```
+   			
+   		var 声明的变量是可以提升的，提升意味着无论变量实际在哪里声明的，都会被当成在当前作用域顶部声明的变量。看下面示例 4。根据示例 1，2，3 显示的情况，建议始终声明变量，无论它们在函数还是全局作用域内。
+        var 声明的函数表达式不能提升。
+
+        对于未声明的变量，可以用 `typeof` 检测其是否存在且不会报错。
+		
+		```js
+          // 示例 1: 声明的变量的作用域在其声明位置的上下文中，而未声明变量是全局的；
```
Comment (zhilidali) [inline js-syntax/02.JavaScript-Expressions and Statements.md]: 未声明变量是全局的？

File: js-syntax/02.JavaScript-Expressions and Statements.md [hunk 2]
```diff
@@ -0,0 +1,1138 @@
...
+
+          const c = 1；
+          c = 2;  // Uncaught SyntaxError: Invalid or unexpected token     
+          ```
+
+   	- ##### 1.2 函数声明
+		> 每个函数都是一个 Function 对象，与其他对象的区别在于可被调用；
+       > 若函数没有 return 语句，则返回 undefined；
+       > 函数是值传递方式（对象是引用传递）。
+       > Es6 开始，严格模式下，块里的函数作用域为这个块。非严格模式下的块级函数不要用。
```
Comment (zhilidali) [inline js-syntax/02.JavaScript-Expressions and Statements.md]: ES6

File: js-syntax/02.JavaScript-Expressions and Statements.md (hunk 1 above)
Comment (YvongYang) [reply js-syntax/02.JavaScript-Expressions and Statements.md]: is anything wrong?

File: js-syntax/02.JavaScript-Expressions and Statements.md (hunk 2 above)
Comment (YvongYang) [reply js-syntax/02.JavaScript-Expressions and Statements.md]: ok. Is anything else?

File: js-syntax/02.JavaScript-Expressions and Statements.md (hunk 1 above)
Comment (zhilidali) [reply js-syntax/02.JavaScript-Expressions and Statements.md]: 应该有限制条件

**Pull request merged**